### ISSUE TITLE
Crew discount for computers should not add up

### ISSUE THREAD

**Issue by Morrus** [herosheets/woin-starships#65]
When multiple control computers are in use, the crew discount %'s should not be added together; instead the best one should be used.

**Comment by jsomara** [comment]
Is this true for multiple _types_ of control computers, or also true for multiple _quantities_ of the same computer?

IE, if I have 10x of Chemerkin-Liang Products LM-2 is my crew discount 10% or 1%?

**Comment by Morrus** [comment]
Just 1%. You only get one discount, once.

**Comment by Morrus** [comment]
This hasn't been implemented. I just created a ship and it applied the discounts every time a computer was added. Relevant section of the stat block below. As you see, I got it to go into negative crew.

It should apply the discount _only once_ no matter how many computers are selected. 

Hull Class XI (INIT -2d6)
Traits 
Crew -400 (cost -80000cr/m); Troops 0; Passengers 0 (0 standard, 0 luxury)
Command & Control Systems
Computers 20x Newstellar EM-1H Command Computer (CPU cycles: 32; max FTL: 8; checks: +1d6)
Sensors none (range 0; check 0)
